platform/linux-generic/odp_ipsec.c

```diff
@@ -1165,6 +1167,8 @@ static int ipsec_out_esp(odp_packet_t *pkt,
 				 ipsec_offset + _ODP_ESPHDR_LEN,
 				 ipsec_sa->esp_iv_len,
 				 state->iv + ipsec_sa->salt_length);
+	_odp_packet_set_data(*pkt, esptrl_offset - esptrl.pad_len - tfc_len,
+			     0xa5, tfc_len);
```

Comment:
Should `0xa5` be a `#define` rather than a "magic number"?

Reply:
Well, it's just a magic number. Same as 0 would be.

Reply:
True, but non-zero numbers tend to stand out when used like this. Why pick this number vs. some other? Is there something wrong with using zeros?

Reply:
I'd still prefer to see this at least commented as to why `0xa5` is used as a pad character.